sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -27,3 +22,84 @@ class UniversalSpinTable(Base):
     # TODO: create table for repointing and then make
     # a foreign key to universal_spin_table
     repointing_number = Column(Integer, nullable=False)
+
+
+class MetadataTable:
+    """Common Metadata table"""
+
+    id = Column(Integer, primary_key=True)
+    file_name = Column(String, nullable=False)
+    # TODO: Do we need instrument since these will be split up into their
+    # separate tables?
+    instrument = Column(String, nullable=False)
+    data_level = Column(String, nullable=False)
+    descriptor = Column(String, nullable=False)
+    start_date = Column(DateTime(timezone=True), nullable=False)
+    end_date = Column(DateTime(timezone=True), nullable=False)
+    ingestion_date = Column(DateTime(timezone=True), nullable=False)
+    version = Column(Integer, nullable=False)
+    format = Column(String, nullable=False)
+
+
+# TODO: Follow-up PR should add in columns for each instrument
+# for instrument dependency IDs
+
+
+class LoMetadataTable(MetadataTable, Base):
+    """IMAP-Lo Metadata Table"""
+
+    __tablename__ = "lo_metadata_table"
```

Comment:
Do we expect any other instrument-specific tables other than metadata tables? If not, I wonder if we should just name the tables `lo`, `hi`, `ultra`, `mag`, etc. It would slightly simplify the queries.

Reply:
I don't think so since we're going to be putting the dependency info in these tables as well, so it would make more sense to not have metadata in the name. I'll make that update.